armi/reactor/reactorParameters.py

```diff
@@ -217,6 +217,13 @@ def detailedNucKeys(self, value):
 
     with pDefs.createBuilder(default=0.0, location="N/A") as pb:
 
+        pb.defParam(
+            "currentGeometryAxialMesh",
+            units="cm",
+            description="Current core axial mesh.",
```

Comment:
Prefer a name that mentions "block" or at least add that to the description.

Reply:
Addressed. https://github.com/terrapower/armi/pull/740/commits/440b6fcab1ee7f95ee42b7d6780ec5cc0fbc540d